gulpfile.babel.js

```diff
@@ -493,14 +495,38 @@ gulp.task( 'languages:extract', function( done ) {
 		} );
 } );
 
+/*
+ * Gutenpack!
+ */
+gulp.task( 'gutenpack', function() {
+	gulp.src( '**/*/*block.jsx' )
```

Comment:
While this is really minor, and we don't do it everywhere, let's `return` the result here so that gulp tracks the task completion properly. Without it we get:
```
[13:59:36] Starting 'gutenpack'...
[13:59:36] Finished 'gutenpack' after 836 μs
```

And when returning the result we get:
```
[14:22:03] Starting 'gutenpack'...
[14:22:05] Finished 'gutenpack' after 2.43 s
```